.github/workflows/crucible-release.yaml

```diff
@@ -0,0 +1,125 @@
+name: crucible-release
+
+on:
+  workflow_call:
+    inputs:
+      dry-run:
+        required: false
+        type: boolean
+        description: Do NOT push new generated tag to the upstream repositories
+        default: false
+
+env:
+  GITHUB_WORKSPACE: ${{ inputs.github_workspace }}
```

Comment:
`inputs.github_workspace` does not exist above

Reply:
good catch